### PR TITLE
Add GET curator/collections

### PR DESCRIPTION
**Pull request by Bento007**
### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

File: backend/open_api_spec.yml [hunk 1]
```diff
@@ -212,16 +266,14 @@ components:
         - visibility
         - links
         - datasets
-        - created_at
-        - updated_at
```
Comment (danieljhegeman) [inline backend/open_api_spec.yml:216]: don't we want to substitute in `revised_at` and `published_at`?

File: backend/open_api_spec.yml [hunk 2]
```diff
@@ -24,8 +24,62 @@ paths:
 
   /curation/collections:
     get:
-      description: Get all Collections metadata
+      description: Get all Collections metadata. Include authentication information to retrieve private collections as well.
```
Comment (danieljhegeman) [inline backend/open_api_spec.yml:27]: This doesn't square with the `security` mechanism. I thought we were restricting all curator api functionality to curators with access tokens.

**Review by danieljhegeman — APPROVED** [review]

File: backend/open_api_spec.yml (hunk 2 above)
Comment (Bento007) [reply backend/open_api_spec.yml:27]: that required field is only used for json that is in a request, not the response.

File: backend/open_api_spec.yml (hunk 1 above)
Comment (Bento007) [reply backend/open_api_spec.yml:216]: that required field is only used for json that is in a request, not the response. So we don't need to put revised and published here.

**Pull request merged**